src/pages/news/news-item/news-item.js

```diff
@@ -15,24 +15,21 @@ import { useTranslation } from 'react-i18next';
 import { useStyles } from './news-item.style';
 import { IMG_URL, TIME_OPTIONS } from '../../../configs';
 
-const NewsItem = ({ date, author, image, title, text, id, slug }) => {
+const NewsItem = ({ date, author, image, title, text, id, slug, translationsKey }) => {
   const { t, i18n } = useTranslation();
-  const language = i18n.language === 'ua' ? 0 : 1;
+  const dateLanguage = i18n.language === 'ua' ? 'ukr-UA' : 'en-US';
   const styles = useStyles();
-  if (!text[language].value) {
+  if (!t(`${translationsKey}.text`)) {
     return null;
   }
-
-  const newsDateLanguageOptions = ['ukr-UA', 'en-US'];
-  const dateLanguage = newsDateLanguageOptions[language];
   return (
     <div className={styles.container}>
       <Card className={styles.root}>
         <div className={styles.imagesContainer}>
           <CardMedia
             className={styles.media}
             image={IMG_URL + image}
-            title={title[language].value || t('newsDetail.noTitle')}
+            title={t(`${translationsKey}.title`) || t('newsDetail.noTitle')}
```

Comment:
Only first condition will be shown because if translation key returns null, text will be null.title so it is true. And for other ternaries same

Reply:
fixed